src/controller/gameController.ts

```diff
@@ -1,47 +1,85 @@
 import { AuthRequest } from './authenticationController.js';
 import {Response, NextFunction} from 'express';
 import { AppError, errorWrapper } from '../utils/errors.js';
+import { v4 as uuidv4 } from 'uuid';
 
 import { LocalStorage } from '../LocalStorage/localStorage.js';
+import { em } from '../database.js';
+import { Game, Player, User } from '../entity/index.js';
+import { Card } from '../LocalStorage/Card.js';
+import { InviteResponse, InviteType } from '../LocalStorage/InviteResponse.js';
+import assert from 'assert';

```

Comment:
Hier wird `assert` von `assert` importiert und einmal wird in `Card.ts` `assert` von `console` importiert. Sollten die imports nicht immer von der gleichen Quelle kommen?